pandas/core/style.py

```diff
@@ -255,6 +272,75 @@ def _translate(self):
                     precision=precision, table_styles=table_styles,
                     caption=caption, table_attributes=self.table_attributes)
 
+    def format(self, formatter, subset=None):
+        '''
+        Format the text display value of cells.
+
+        .. versionadded:: 0.18.0
+
+        Parameters
+        ----------
+        formatter: str, callable, or dict
+        subset: IndexSlice
+
+        Returns
+        -------
+        self: Styler
```

Comment:
I think maybe update the other doc-strings (``.apply``, ``.applymap``) etc to the same ``Returns`` (e.g. ``self : Styler``) is pretty clear